### PR TITLE
status codes are useful if you want your templates to render generically

### PR DESCRIPTION
**Pull request by kode4food**

### CODE REVIEW

File: middleware/404.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
 module.exports = function (template) {
 
     return function fileNotFound(req, res, next) {
-        var model = { url: req.url };
+        var model = { url: req.url, statusCode: 404 };
```
Comment (kode4food) [inline middleware/404.js:24]: statusCode: 404 will find its way into the 404.dust template

File: middleware/500.js [hunk 2]
```diff
@@ -21,7 +21,7 @@
 module.exports = function (template) {
 
     return function serverError(err, req, res, next) {
-        var model = { url: req.url, err: err };
+        var model = { url: req.url, err: err, statusCode: 500 };
```
Comment (kode4food) [inline middleware/500.js:24]: statusCode: 500 will find its way into the 500.dust template

**Comment by kode4food** [comment]
this change shouldn't break any existing code unless someone had been implementing an error template that absolutely relied on statusCode /not/ being there.  Any chance it will be pulled soon?

**Comment by totherik** [comment]
Looks good to me. :+1:

**Comment by grawk** [comment]
Merging now

**Comment by kode4food** [comment]
Thanks much!

**Pull request merged**